### ISSUE TITLE
Redundancy analysis

### ISSUE THREAD

**Issue by goodtimeaj** [CSSLint/csslint#415]
Could csslint have built-in redundancy analysis like the functionality of [https://github.com/zmoazeni/csscss](https://github.com/zmoazeni/csscss)? As far as I know, csslint doesn't warn about cases like this:

``` css
.module-1 {
  font-size: 14px;
}

.module-1 .module-2 {
  font-size: 14px;
}
```

Is this within the scope of the project? Should it be?
